omod/src/test/java/org/openmrs/module/registrationapp/fragment/controller/RegisterPatientFragmentControllerTest.java

```diff
@@ -43,10 +46,9 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
 
+import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.core.Is.is;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
We tend to avoid wildcard imports :)

Reply:
Ugh, sorry.  Dumb default Intellij setting on a new project.